lib/getData.py

```diff
@@ -0,0 +1,103 @@
+import os
+import fire
+import shutil
+
+import numpy as np
+import astroquery.mast as mast
+import astropy.io.fits as pyfits
+
+def getData (pi, target, pid, sciFil, imgFil, fileSearch='ima.fits', local='.', sortPath='.'):
+    """
+    Description
+    ----------
+    Downloads data from MAST for a given pi, target, and proposal id, and sorts into directories for WFC3.
+
+    Parameters
+    ----------
+    pi: string
+        Name of pi, wild cards are allowed
+    target: string
+        Name of target star
+    pid: string
+        Proposal id number.
+    sciFil: string
+        Filter used for science observations
```

Comment:
This description is a little vague.  Maybe provide a list of possible values (G102, G141, G*, etc).  Same with `imgFil` (F*).  Do wildcards work here?

Reply:
Alternatively, the code could just check the header to see if it's an F or G.  Then these two parameters would be unnecessary.

Reply:
Technically the filter can be anything that mast could possibly return you.... Wild cards do not work here as written, but we could make them work here.

Reply:
I think we should just query the header keywords `OBSTYPE` to determine if a file is spectroscopic or imaging.  That negates the need for these two parameters.